Sources/Purchasing/Configuration.swift

```diff
@@ -143,12 +151,26 @@ import Foundation
          * RevenueCat's backend. Default is `false`.
          *
          * - Warning: This assumes your IAP implementation uses StoreKit 1.
-         * Observer mode is not compatible with StoreKit 2.
+         * `.myApp` is not compatible with StoreKit 2.
```

Comment:
I think this will end up changing when this gets rebased by `5.0-dev` branch, right? 🤔  cc: @fire-at-will @MarkVillacampa

Reply:
Yep we'll change this when it gets rebased.

You can keep it, that way we'll surely notice when we get a conflict 😄